roles/acm_setup/README.md

```diff
@@ -42,6 +46,29 @@ See below an example of how to use the acm_setup role to configure ACM.
       hub_disable_selfmanagement: true
       hub_availabilityConfig: High
       hub_disconnected: true
+      hub_os_images:
+        - openshiftVersion: "4.15.0-0.nightly-2024-04-21-051624"
+          version: "4.15"
+          url: "http://webcache.dfwt5g.lab:8080/rhcos-415.92.202402201450-0-live.x86_64.iso"
+          cpuArchitecture: x86_64
+```
+
+## OS images
+
+```yaml
+hub_os_images:
+  - cpuArchitecture: x86_64
+      openshiftVersion: "4.<NN>"
+      rootFSUrl: https://<host>/<path>/rhcos-live-rootfs.x86_64.img
+      url: https://<mirror-registry>/<path>/rhcos-live.x86_64.iso
+  - cpuArchitecture: x86_64
+      openshiftVersion: "4.<NN>"
+      url: https://<mirror-registry>/<path>/rhcos-4.14-live.x86_64.iso
```

Comment:
```suggestion
      url: https://<mirror-registry>/<path>/rhcos-4.<minor-version>-live.x86_64.iso
```

Reply:
Fixed there and in other places. Thanks!